src/thor/worker.cc

```diff
@@ -121,44 +121,45 @@ namespace valhalla {
 
         worker_t::result_t result{true};
         double denominator = 0;
-        size_t order_index = 0;
```

Comment:
all the changes in here are to reserialize the pbf request when you make changes on it so that once it gets to `odin` it has those changes reflected